tests/test_timerange.py

```diff
@@ -716,6 +716,59 @@ def test_normalise(self):
                                  msg=("{!r}.normalise({}, {}, rounding={}) == {!r}, expected {!r}"
                                       .format(tr, rate.numerator, rate.denominator, rounding, result, expected)))
 
+    def test_is_normalise(self):
+        tests_tr = [
```

Comment:
NIT: This was copied from the normalise test. It might be worth moving the array elsewhere so it is re-used rather than duplicated. Alternatively, the expected result here is the true/false output. So you could replace the second TimeRange in the data set with True/False as required and assertEqual against that.